arklex/orchestrator/entities/orchestrator_state_entities.py

```diff
@@ -214,6 +214,8 @@ class OrchestratorState(BaseModel):
     message_flow: str = Field(default="")
     # record history
     function_calling_trajectory: list[dict[str, Any]] | None = Field(default=None)
+    # OpenAI Agents SDK specific trajectory (user/assistant/messages and tool records)
+    openai_sdk_trajectory: list[dict[str, Any]] | None = Field(default=None)
```

Comment:
can we define the type instead of using dict? Maybe we can directly import from agents sdk?